html/index.html

```diff
@@ -0,0 +1,480 @@
+                </p>
+        
+              </div>
+        
+              <div class="footer-third-container">
+                
+                <p><a href="#">LET'S CONNECT</a> </p>
+        
+              </div>
+        
+              <div class="lower-footer">
+        
+                <img src="../assets/twitter.svg" alt="">
+                <img src="../assets/linkdn.svg" alt="">
+                <img src="../assets/medium.svg" alt="">
+                <img src="../assets/github.svg" alt="">
+                <img src="../assets/rabbit.svg" alt="">
+        
+              </div>
+    
+              <div class=" top-footer-btn">
+                <button id="third-container-button">Get my resume</button>
+              </div>
+        
+            </div>
+
+            <div class="languages-container">
+    
+                <div class="language-container-top">
+        
+                    <p class="bold-text">Languages</p>
+        
+                    <img src="../assets/dropdown.png" alt="" id="dropdown-img">
+        
+                </div>
+        
+                <div class="language-tools">
+        
+                    <div class="language-java-script">
+        
+                        <img src="../assets/jsLogo.png" alt="">
+        
+                        <div id="javascript">
+                            
+                            <span> JavaScript</span>
+        
+                        </div>
+        
+                    </div>
+        
+                    <div class="language-html">
+        
+                        <img src="../assets/h5Logo.png" alt="">
+        
+                        <div id="html">
+                            
+                            <span> HTML</span>
+        
+                        </div>
+        
+                    </div>
+        
+                    <div class="language-css">
+        
+                        <img src="../assets/css3Logo.png" alt="">
+        
+                        <div id="css"> 
+                            
+                            <span> CSS </span>
+        
+                        </div>
+        
+                    </div>
+        
+                </div>
+        
+                <div class="language-footer">
+        
+                    <div class="language-framwork">
+        
+                        <p class="bold-text">Frameworks</p>
+        
+                        <img src="../assets/dropdown.png" alt="" class="dropside-img">
+            
+                    </div>
+        
+                    <hr id="hr">
+        
+                    <div class="language-skill">
+        
+                        <p  class="bold-text">Skills</p>
+        
+                        <img src="../assets/dropdown.png" alt="" class=" dropside-img">
+            
+                    </div>
+        
+                    <hr>
+        
+        
+        
+                </div>
+        
+        
+            </div>
+
```

Comment:
- [ ] Same as here:
- The technology categories ("Languages", "Frameworks" and "Skills") are in a `ul`, each of them in a `li`
- The technology in each category(e.g., "JavasScript", "HTML" and "CSS") is also a list `ul` and each of them a `li`

Reply:
- @joskalenda as I suggested [above](https://github.com/Simpleshaikh1/MyMicroversePortfolio/pull/1#discussion_r1149180908) that section isn't part of this milestone so please make sure that @Simpleshaikh1 removes it altogether.

Reply:
- [ ] hey @AbelG101, you were right, and think was right too, in your comment you say the student should remove the contact form part I think, which is correct on the finished mobile version milestone, the contact form is not included, and in my case, I told the student to fix the style of the `framework and language section`, not the contact form, which is included in the finish mobile version milestone, please check well where the comment ends

Reply:
thanks, @AbelG101